neural_lam/train_model.py

```diff
@@ -20,36 +23,28 @@
 }
 
 
+@logger.catch
 def main(input_args=None):
-    """
-    Main function for training and evaluating models
-    """
+    """Main function for training and evaluating models."""
     parser = ArgumentParser(
         description="Train or evaluate NeurWP models for LAM"
     )
     parser.add_argument(
-        "--data_config",
+        "config",
```

Comment:
ok, we can do that. But what should default value then be? I changed this because I am not sure it makes sense to have a default unless we have a default configuration and datastore that people should use?

Reply:
No I don't think a default makes sense. `assert args.config is not None, "Need to specify a config ..."`

Reply:
I've tried to resolve this in https://github.com/mllam/neural-lam/pull/66/commits/772cc203f8e22f3f3f4374e368df85fb6455ed72. Feel free to resolve this if you are happy with that implementation :)

Reply:
Why does that commit add in the `--datastore_kind` flag again?

Reply:
> Why does that commit add in the --datastore_kind flag to the plotting script? Is that not read from the config? It seems unused.

That is because it is taking the datastore config as an argument, not the neural-lam config. I could change that so that the neural-lam config is given instead. But it might be nice to keep this as-is so that when we in future have multiple datastores to plot from.

Reply:
I see, yes that makes sense. I was mostly confused since it was not used later. I think I fixed that script now though with https://github.com/mllam/neural-lam/pull/66/commits/665368dfb8485ae93857ee090229f4d93a20f071 . I was not able to test it properly though, since the default arguments don't seem to work for me locally to produce any plot. Maybe you can try it out and see if it works for you?

Reply:
Ok, great! Then this is also resolved.